src/pages/create-wiki/index.tsx

```diff
@@ -261,7 +261,9 @@ const CreateWikiContent = () => {
       // Build the wiki object after edit info has been calculated
       const finalWiki = {
         ...interWiki,
-        metadata: store.getState().wiki.metadata,
+        metadata: store.getState().wiki.metadata.filter(meta => {
+          return meta.value !== '' || meta.id === 'references'
```

Comment:
the references in string can use enum 
```suggestion
          return meta.value !== '' || meta.id === CommonMetaIds.REFERENCES
```